app/views/replies/_single.haml

```diff
@@ -1,7 +1,8 @@
-- if reply.is_a?(Post)
-  - write = 'post-post'.freeze
-- elsif reply.is_a?(Reply)
-  - write = 'post-reply'.freeze
+- if reply.is_a?(Reply)
```

Comment:
This should _only_ take a reply – especially given that, on line 16, we have an unguarded reference to `reply.icon_id`, which won't exist on a post. Right? (See also <https://github.com/Marri/glowfic/pull/705#issuecomment-513939874>)